principles/lawOfDemeter/q-essay-demeter.md

```diff
@@ -7,14 +7,14 @@ Explain the Law of Demeter using code examples. You are to make up your own code
 
 Let us take the `Logic` class as an example.  Assume that it has the following operation.
 
-`setMinefield(Minefield mf):void`
+`setMinefield(Minefield mf): void`
 
-Consider the following that can happen inside this operation.
+Consider the following that can happen inside this operation:
 
 * `mf.init();`: this does not violate LoD since LoD allows calling operations of parameters received.
-* `mf.getCell(1,3).clear();`: //this violates LoD because `Logic` is handling `Cell` objects deep inside `Minefield`. Instead, it should be `mf.clearCellAt(1,3);`
-* `timer.start();`: //this does not violate LoD because `timer` appears to be an internal component (i.e. a variable) of `Logic` itself.
-* `Cell c = new Cell();` `c.init();`: // this does not violate LoD because `c` was created inside the operation.
+* `mf.getCell(1,3).clear();`: this violates LoD because `Logic` is handling `Cell` objects deep inside `Minefield`. Instead, it should be `mf.clearCellAt(1,3);`.
```

Comment:
```suggestion
* `mf.getCell(1, 3).clear();`: this violates LoD because `Logic` is handling `Cell` objects deep inside `Minefield`. Instead, it should be `mf.clearCellAt(1, 3);`.
```